### DISCUSSION TITLE
Is it suitable for use in Production?

### DISCUSSION THREAD

**Discussion by Foixa** [hmlongco/Factory#65] [General]
Should I use Factory in production? Which version(1+ or 2+) would you recommend for now?

**Comment by hmlongco** [comment]
Factory has been in use in production applications for about a year now. If you can wait about a week 2.0 is going to drop and you can save a bit of effort in not converting your application from 1.0 to 2.0.

I should also note that Factory 2.0 passes the same unit tests with the same code coverage as 1.0. I've been using it myself and wouldn't have any qualms recommending its use.

**Comment by doozMen** [comment]
we use verion 1.0 after using Resolver for several years now in production and works very nice. Looking forward wot 2.0!
